contracts/exchange/test/internal.ts

```diff
@@ -48,16 +50,358 @@ blockchainTests('Exchange core internal functions', env => {
     let testExchange: TestExchangeInternalsContract;
     let logDecoder: LogDecoder;
     let senderAddress: string;
+    let makerAddressLeft: string;
+    let makerAddressRight: string;
+    let orderFactoryLeft: OrderFactory;
+    let orderFactoryRight: OrderFactory;
 
     before(async () => {
```

Comment:
This makes me sad :panda_face: @ how this file ballooned by 6X, and so much of it is visually similar. There must be a way to cut this down.

Reply:
I'll look into it. I think that creating a helper to reduce the size of the `expectedMatchedFillResults` objects could help with this.

Reply:
It also might make sense to start splitting `internal` up. I think that reducing the code in this PR is a fair point, but it's going to be pretty big even after some refactoring. I'd even go so far as to say that we could do a unit test file per significant function (like a `calculateFillResults` or `calculateMatchedFillResults`). We could keep internal around just for the smaller or easier to test functions.